nuxt.config.js

```diff
@@ -1,87 +1,80 @@
-/** @type {import('highcharts/highcharts').Options} */ 
-const setOptions = { 
-  lang: { // <-- we correctly get intellisense
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
+/** @type {import('highcharts/highcharts').Options} */
+const setOptions = {
+  lang: {
+    // <-- we correctly get intellisense
     // decimalPoint: '.' // this can be changed to anything, like ',' and it works
   }
-}
+};
 
 /** @type {import('highcharts/highcharts').Options} */
-const chartOptions = { // <-- we also get intellisense correctly. 
+const chartOptions = {
+  // <-- we also get intellisense correctly.
   credits: {
     enabled: true // <-- typing "e" suggests "enabled?" ok
   }
-}
+};
 // Important note: as of 12/2020 it seems...
 // per the API docs, the "setOptions" get passed to the "Highcharts.setOptions" method
 // while the chartOptions would get passed to the *Highcharts.chart* method
 
-module.exports = {
-  env: {},
-  mode: 'spa', // 'universal', // TBD: use highcharts *src* for ssr mode?
-  /*
-   ** Headers of the page
-   */
+export default {
+  // Global page headers (https://go.nuxtjs.dev/config-head)
   head: {
-    title: process.env.npm_package_name || '',
+    title: "highcharts",
```

Comment:
I think this should be reverted back to process.env.npm_package_name. This one of those minor things that's easy to miss as we go from project to project. This will be one less chore for us in the future.

Reply:
will do